css/properties/image-rendering.json

```diff
@@ -198,22 +198,22 @@
                 "version_added": false
               },
               "opera": {
-                "version_added": true
+                "version_added": "15"
               },
               "opera_android": {
-                "version_added": null
+                "version_added": "14"
               },
               "safari": {
-                "version_added": true
+                "version_added": "6.1"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "7"
               },
               "samsunginternet_android": {
-                "version_added": false
+                "version_added": "1.5"
               },
               "webview_android": {
-                "version_added": false
+                "version_added": "41"
```

Comment:
41 here (and on the other WebView entry below) is a bit surprising. Based on everything else, I expected to see 4.4 or ≤37. Is there a story behind this?

Reply:
Oh, yes, I forgot to mention this. I've set this to 41, as that is what the parent feature `image-rendering` says for `webview_android`.

Reply:
Good enough!